frontend/components/TableContainer/DataTable/StatusCell/StatusCell.tsx

```diff
@@ -13,12 +14,42 @@ const generateClassTag = (rawValue: string): string => {
 };
 
 const StatusCell = ({ value }: IStatusCellProps): JSX.Element => {
+  console.log(`value: ${value}`);
   const statusClassName = classnames(
     "data-table__status",
     `data-table__status--${generateClassTag(value)}`
   );
+  const tipContent = (value: string): string | undefined => {
+    switch (value) {
+      case "online":
+        return "Online hosts will respond to a live query.";
+      case "offline":
+        return `Offline hosts won't respond to a live query because<br/>
+                they may be shut down, asleep, or not connected to<br/>
+                the internet.`;
```

Comment:
if `online` and `offline` are the only two values this can be a single if conditional

```ts
if (value === "online") { return ... }

return ...
```